pkg/controller/master/upgrade/job_controller.go

```diff
@@ -0,0 +1,119 @@
+package upgrade
+
+import (
+	"reflect"
+
+	v1 "k8s.io/api/core/v1"
+
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/labels"
+
+	apisv1alpha1 "github.com/rancher/harvester/pkg/apis/harvester.cattle.io/v1alpha1"
+	"github.com/rancher/harvester/pkg/generated/controllers/harvester.cattle.io/v1alpha1"
+	upgradev1 "github.com/rancher/harvester/pkg/generated/controllers/upgrade.cattle.io/v1"
+)
+
+const (
+	stateUpgrading     = "Upgrading"
+	stateSucceeded     = "Succeeded"
+	stateFailed        = "Failed"
+	helmChartLabel     = "helmcharts.helm.cattle.io/chart"
+	upgradePlanLabel   = "upgrade.cattle.io/plan"
+	upgradeNodeLabel   = "upgrade.cattle.io/node"
```

Comment:
are those labels specified by the system-upgrade-controller?

Reply:
Yes